andy/src/main/java/nl/tudelft/cse1110/andy/result/CodeChecksResult.java

```diff
@@ -21,11 +21,21 @@ public static CodeChecksResult build(List<CodeCheckResult> checkResults) {
     }
 
     public int getNumberOfPassedChecks() {
-        return checkResults.stream().mapToInt(check -> check.passed() ? check.getWeight() : 0).sum();
+        return getNumberOfPassedChecks(true);
+    }
+
+    public int getNumberOfPassedChecks(boolean includeWeight) {
```

Comment:
One way to avoid this boolean would be to offer two methods `weightedNumberOfPassedChecks`. 

See this blog post explaining this Clean Code rule: https://www.informit.com/articles/article.aspx?p=1392524

(This isn't blocking. The refactoring doesn't need to happen in this MR!)

Reply:
I made this into a separate issue: #219 

This should be an easy refactoring and perhaps a student might find it interesting :)